index.html

```diff
@@ -89,16 +118,34 @@ <h2>Tonic</h2>
       <img src="images/Action6.png" alt="button images">
      </div>
    </div>
+  -->
 </div>
 <div class="cardwork4" >
+  <div class="card4left">
+    <img src="images/card4left.png" alt="iamges">
+  </div>
+  <div class="card4right">
+    <img src="images/card4right.png" alt="images">
+  </div>
+  <!---
   <div class="portfolio6">
     <img src="images/SnapshootPortfolio6.png" alt="portfolio images">
   </div>
   <div class="leftblock6">
     <img src="images/LeftBlock6.png" alt="blocks images">
   </div>
```

Comment:
- [ OPTIONAL] PLease remove unused codes in your project for a good practice so that your project will look professional :wink:  kindly remove them in all your projec :upside_down_face: